src/R/Components/Impl/InteractiveWorkflow/Implementation/InteractiveWindowConsole.cs

```diff
@@ -17,19 +17,28 @@ public sealed class InteractiveWindowConsole : IConsole {
             _workflow = workflow;
         }
 
+        public void WriteError(string text) {
+            WriteAsync(text, true).DoNotWait();
+        }
+
         public void Write(string text) {
```

Comment:
I call WriteLine from SendFileCommandBase to write our the local and remote paths.